File: SDKs/Aspose.Imaging_Cloud_SDK_for_Java/src/main/java/com/aspose/imaging/model/TiffProperties.java
package com.aspose.imaging.model;

import com.aspose.imaging.model.ExifData;
public class TiffProperties {
  private String ByteOrder = null;
  private ExifData ExifData = null;
  /**
         * getByteOrder
         * Gets String
         * @return ByteOrder
         */
  public String getByteOrder() {
    return ByteOrder;
  }

        /**
         * setByteOrder
         * Sets String
         * @param ByteOrder String
         */
  public void setByteOrder(String ByteOrder) {
    this.ByteOrder = ByteOrder;
  }

  /**
         * getExifData
         * Gets ExifData
         * @return ExifData
         */
  public ExifData getExifData() {
    return ExifData;
  }

        /**
         * setExifData
         * Sets ExifData
         * @param ExifData ExifData
         */
  public void setExifData(ExifData ExifData) {
    this.ExifData = ExifData;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TiffProperties {\n");
    sb.append("  ByteOrder: ").append(ByteOrder).append("\n");
    sb.append("  ExifData: ").append(ExifData).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Imaging_Cloud_SDK_for_CSharp/src/Com/Aspose/Imaging/Model/TiffProperties.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Imaging.Model {
  public class TiffProperties {
    public string ByteOrder { get; set; }

    public ExifData ExifData { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TiffProperties {\n");
      sb.Append("  ByteOrder: ").Append(ByteOrder).Append("\n");
      sb.Append("  ExifData: ").Append(ExifData).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
